tests/xilinx/macc.ys

```diff
@@ -0,0 +1,31 @@
+read_verilog macc.v
+design -save read
+
+proc
+hierarchy -top macc
+#equiv_opt -assert -map +/xilinx/cells_sim.v synth_xilinx ### TODO
```

Comment:
Why doesn't his work?!

Reply:
I expect this is a problem with the clock enable meaning there are combinations of inputs for which the outputs of gate and gold will never synchronise, causing induction to fail

Reply:
Please forgive my ignorance, but can you elaborate on this a little? Any clock enable input would be fed equally to gate and gold, which would be expected to implement identical behaviour, no?

Reply:
The initial state will be random and not necessarily the same between gate and gold. Induction checks that, given the output is the same for K cycles, it will be the same at cycle K+1. As one possible input sequence is CE low for those K cycle and high on K+1 (i.e. so the state between the two never synchronises); induction will not pass

Reply:
I see. That makes a lot of sense, thanks! Is there any way to guide induction to throw a CE transition or two in there as part of those K cycles, and whether that would help things at all?